From: Fuel OPNFV <fuel@opnfv.org>
Date: Wed, 3 Aug 2016 09:41:21 +0200
Subject: [PATCH] Mark Intel 82599 10 Gigabit NIC as DPDK capable

This NIC is used on Ericsson POD2 but Fuel doesn't allow to activate
DPDK on it.
---
 nailgun/nailgun/fixtures/openstack.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nailgun/nailgun/fixtures/openstack.yaml b/nailgun/nailgun/fixtures/openstack.yaml
index 74fa509..d08a4fc 100644
--- a/nailgun/nailgun/fixtures/openstack.yaml
+++ b/nailgun/nailgun/fixtures/openstack.yaml
@@ -2213,7 +2213,7 @@
           "15ad:07b0", "8086:15a5", "1137:0043", "1137:0071", "14e4:168a",
           "14e4:16a9", "14e4:164f", "14e4:168e", "14e4:16af", "14e4:163d",
           "14e4:163f", "14e4:168d", "14e4:16a1", "14e4:16a2", "14e4:16ad",
-          "14e4:16ae", "14e4:163e", "14e4:16a4"
+          "14e4:16ae", "14e4:163e", "14e4:16a4", "8086:10f8"
         ]
 - pk: 3
   extend: *ubuntu_release
